apps/styleguide/src/display/LinksList.vue

```diff
@@ -32,12 +32,19 @@ export default {
 <docs>
 Display a list of links
 ```
-const links = [
-    { label: 'Molgenis', href: 'https://molgenis.org', },
-    { label: 'Vue',  href: 'https://vuejs.org/' } 
-]
+<script>
+  export default {
+    data() {
+      return {
+        links: [{label: 'Molgenis', href: 'https://molgenis.org'}]
+      }
+    }
+  }
+</script>
 <template>
-  <links-list :items="links" ><p></p>Sample content</p></<links-list>
+  <LinksList :items="links">
```

Comment:
lower case no is the html tag default / suggest style , suggest to keep it

Reply:
hmm, I am not a fan of that. But changed it to make you happy :-)